src/main/java/in/projecteka/consentmanager/user/model/IdentifierType.java

```diff
@@ -16,4 +18,15 @@ public boolean isValid(String abpmJayValue) {
     };
 
     public abstract boolean isValid(String value);
+
+    public static final HashMap<IdentifierType, IdentifierGroup> identifierTypeGroupMap = new HashMap<>() {
```

Comment:
I would have preferred to keep this mapping external. You already have a Utils class, do you want to move it there? In the future, this mapping should be externalized.

Reply:
moved to IdentifierUtils